### PR TITLE
Implements front end raw review rendering based of url '/:id'

### PR DESCRIPTION
**Pull request by matt-wigg**

### CODE REVIEW

File: client/index.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import ReactDOM from 'react-dom';
+import axios from 'axios';
+import ReviewsRender from './components/ReviewsRender';
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      reviews: [],
+      reviewCount: 3,
```
Comment (miraclestyle) [inline client/index.jsx:11]: @Matt-Wigg , what does reviewCount do? Changes the number of reviews rendered in the component? 

If that's the case, how many reviews do you fetch from the backend in a single request? And do you have a pagination mechanism to fetch more reviews once all reviews on the client side are rendered?

**Review by miraclestyle — COMMENTED** [review]
@Matt-Wigg, take a look at the comment. I am interested to know more about your approach. Everything else looks ok.

File: client/index.jsx (hunk 1 above)
Comment (matt-wigg) [reply client/index.jsx:11]: Hey @miraclestyle - review count does exactly that! It's a way for me to specify the amount of reviews displayed at one time. I pull all reviews related to a particular product Id from my initial get request, then do the filtering client side.

**Comment by matt-wigg** [comment]
Thanks for the review Elvin - I'm going to merge in to continue!

File: client/index.jsx (hunk 1 above)
Comment (miraclestyle) [reply client/index.jsx:11]: I understand. This will, perhaps, be a good opportunity to work on during the optimization phase.

**Pull request merged**